main.py

```diff
@@ -1,10 +1,68 @@
 import init_django_orm  # noqa: F401
 
+import json
+from datetime import datetime
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_nickname, player in players_data.items():
+
+        # Creating a race
+
+        race_name = player.get("race").get("name")
+        race_description = player.get("race").get("description")
+        if not Race.objects.filter(
+                name=race_name).exists() and race_description:
+            race_obj = Race.objects.create(
+                name=race_name,
+                description=race_description
+            )
+        elif not Race.objects.filter(
+                name=race_name).exists() and race_description is None:
+            race_obj = Race.objects.create(name=race_name)
```

Comment:
3 queries always as minimum. You can use get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create
Fix in other places

Reply:
Great method, thanks!